src/dbFirebase.js

```diff
@@ -83,15 +82,18 @@ async function fetchPhotos() {
 
 function saveMetadata(data) {
   data.location = new firebase.firestore.GeoPoint(data.latitude, data.longitude);
-  data[config.PHOTO_FIELD.name]= config.PHOTO_FIELD.type === enums.TYPES.number ? Number(data.field) : data.field
+  delete data.latitude;
+  delete data.longitude;
 
   if (firebase.auth().currentUser) {
     data.owner_id = firebase.auth().currentUser.uid;
   }
   data.updated = firebase.firestore.FieldValue.serverTimestamp();
   data.moderated = null;
 
-  const fieldsToSave = [config.PHOTO_FIELD.name, "moderated", "updated", "location", "owner_id"];
+  let fieldsToSave = ["moderated", "updated", "location", "owner_id"];
+  Object.values(config.PHOTO_FIELDS).forEach(field => fieldsToSave.push(field.name));
```

Comment:
is this a `_.map` ?

Reply:
I think it's not because I want to add extra fields on the fieldsToSave array.